src/orange/orangeinp/RevolvedPolygon.cc

```diff
@@ -62,7 +66,23 @@ NodeId RevolvedPolygon::build(VolumeBuilder& vb) const
 
     // Start the recursion process
     SubIndex ri;
-    return this->make_levels(vb, filtered_polygon, ri);
+    auto result = this->make_levels(vb, filtered_polygon, ri);
+
+    if (enclosed_)
+    {
+        // Handle azimuthal truncation
+        auto&& [sense, wedge] = enclosed_.make_sense_region();
+        NodeId wedge_id = build_intersect_region(vb, label_, "angle", wedge);
+        if (sense == Sense::outside)
+        {
+            wedge_id = vb.insert_region(Label{label_, "negated_angle"},
```

Comment:
To be consisted with Solid (and actually could you update PolySolidBase L97-98 too?)
```suggestion
        char const* ext = (sense == Sense::outside ? "~azi" : "azi");
        NodeId wedge_id
            = build_intersect_region(vb, label_, ext, wedge);
        if (sense == Sense::outside)
        {
            wedge_id = vb.insert_region({}, Negated{wedge_id});
```